SPECS/ocaml-bisect-summary.spec

```diff
@@ -0,0 +1,40 @@
+
+Name:           ocaml-bisect-summary
+Version:        0.4
+Release:        1%{?dist}
+Summary:        Report code coverage from bisect_ppx runtime files
+License:        LGPL
+URL:            https://github.com/lindig/bisect-summary
+Source0:        https://github.com/lindig/%{name}/archive/%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  oasis-devel
```

Comment:
Assuming the output of the oasis setup step has been checked in, `-devel` is no longer needed to build the package?